src/pages/Dashboard/PieChart.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useRef, useState } from 'react';
+import ApexCharts from 'apexcharts';
+import { Box } from '@chakra-ui/react';
+
+interface User {
+  id: number;
+  name: string;
+}
+
+const ChartComponent = () => {
+  const [data, setData] = useState<number[]>([]);
```

Comment:
Nao coloque strings magicas no codigo, use o que esta no .env

```suggestion
  const host  = import.meta.env.VITE_API_HOST;
  const [data, setData] = useState<number[]>([]);
```